genai-perf/genai_perf/parser.py

```diff
@@ -499,6 +507,16 @@ def _infer_prompt_source(args: argparse.Namespace) -> argparse.Namespace:
     return args
 
 
+def _infer_tokenizer(args: argparse.Namespace) -> argparse.Namespace:
```

Comment:
This logic needs to be replicated in the config as well otherwise what we do with the CLI isn't consistent with the config file.

Reply:
Done!